index.html

```diff
@@ -834,201 +805,19 @@ <h3>
         </section>
       </section>
     </section>
-    <section>
```

Comment:
This is now https://github.com/whatwg/html/pull/5581 ...

Reply:
Maybe I missed this, but will the section of the HTML specification which defines this be referenced from the Web App Manifest specification so that implementers know where to look? (Maybe just not yet because the reference doesn't exist yet!)

Reply:
Yes, once https://github.com/whatwg/html/pull/5581 gets merged, we will update accordingly.